packages/lastrev-build-prefetch-content/src/helpers/compose.ts

```diff
@@ -54,6 +54,7 @@ const compose = ({
             return v;
           }
           if (k === 'fields') {
+            if (Array.isArray(v)) return v;
```

Comment:
Why is this added? Looks like this would not do the compose on items in an array.